packages/eslint-plugin/docs/rules/no-duplicate-type-constituents.mdx

```diff
@@ -18,7 +18,7 @@ We consider types to be duplicate if they evaluate to the same result in the typ
 For example, given `type A = string` and `type T = string | A`, this rule would flag that `A` is the same type as `string`.
 
 This rule also disallows explicitly listing `undefined` in a type union when a function parameter is marked as optional.
-Doing so is unnecessary.
+Doing so is unnecessary. Please note that this rule only applies to parameters, not properties. Therefore, it does not conflict with the [`exactOptionalPropertyTypes`](https://www.typescriptlang.org/tsconfig/#exactOptionalPropertyTypes) TypeScript compiler setting.
```

Comment:
Two nits
1. The style in this codebase is to use line breaks for new sentences 🤷 
2. I suggest changing "this rule" to "this check", since this note is just about the specific check of function parameters, not the rule as a whole.

```suggestion
Doing so is unnecessary. 
Please note that this check only applies to parameters, not properties. 
Therefore, it does not conflict with the [`exactOptionalPropertyTypes`](https://www.typescriptlang.org/tsconfig/#exactOptionalPropertyTypes) TypeScript compiler setting.
```

What do you think?

Reply:
Both changes make total sense and I've updated the PR accordingly. Thanks!